### PR TITLE
BaseVertex BaseInstance Conformance Test

### PR DESCRIPTION
**Pull request by shrekshao**
Add conformance test for new extensions: WEBGL_draw_instanced_base_vertex_base_instance and WEBGL_multi_draw_instanced_base_vertex_base_instance.

Referencing #2937

### CODE REVIEW

File: sdk/tests/conformance2/extensions/webgl-multi-draw-instanced-base-vertex-base-instance.html [hunk 1]
```diff
@@ -0,0 +1,717 @@
...
+      wtu.glErrorShouldBe(gl, gl.NO_ERROR, "there should be no errors");
+
+      checkResult(config);
+    }
+
+    checkDraw({
+      drawFunc: multiDraw ? multiDrawArraysInstancedBaseInstance : drawArraysInstancedBaseInstance,
+      isDrawArrays: true,
+      isMultiDraw: multiDraw,
+      useBaseVertexBuiltin: true,
```
Comment (kenrussell) [inline sdk/tests/conformance2/extensions/webgl-multi-draw-instanced-base-vertex-base-instance.html]: Do you plan to add tests with useBaseVertexBuiltin set to false? If so, could you add a TODO about that?

**Comment by kenrussell** [comment]
James / Austin - could you both take a look at these new tests as well?

File: sdk/tests/conformance2/extensions/webgl-multi-draw-instanced-base-vertex-base-instance.html (hunk 1 above)
Comment (shrekshao) [reply sdk/tests/conformance2/extensions/webgl-multi-draw-instanced-base-vertex-base-instance.html]: Now added all 4 combinations of useBaseVertexBuiltin/useBaseInstanceBuiltin

File: sdk/tests/conformance2/extensions/webgl-multi-draw-instanced-base-vertex-base-instance.html [hunk 2]
```diff
@@ -0,0 +1,717 @@
...
+  function getShaderSource(countX, countY, config) {
+    const vs = [
+      '#version 300 es',
+      config.isMultiDraw ? '#extension GL_ANGLE_multi_draw : require' : '',
+      '#extension GL_ANGLE_base_vertex_base_instance : require',
+      '#define kCountX ' + countX.toString(),
+      '#define kCountY ' + countY.toString(),
+      'layout(location = 0) in vec2 vPosition;',
+      config.useBaseInstanceBuiltin ? '' : 'layout(location = 1) in float vInstanceID;',
+      !config.isDivisorTest ? '' : 'layout(location = 2) in float vInstanceColorID;', //?? TODO: complete test adjustment
```
[the remark this answers is not in the document]
Comment (shrekshao) [reply sdk/tests/conformance2/extensions/webgl-multi-draw-instanced-base-vertex-base-instance.html]: Removed. This is for some level of angle implementation test and I don't think it needed for here. (2 instanced arrays with different divisor)

**Comment by shrekshao** [comment]
I will merge and roll webgl conformance first to land some chromium patch. Feel free to add comments.

**Comment by kenrussell** [comment]
Sounds good. Again, nice work on these tests.

**Pull request merged**